config/config.exs

```diff
@@ -1,4 +1,2 @@
 use Mix.Config
 
-config :bugsnag,
```

Comment:
Yeah, I guess it is – sorry it's been a long day and I was just looking at the aggregated changes view.